control-plane-operator/controllers/hostedcontrolplane/storage/assets/10_deployment-hypershift.yaml

```diff
@@ -88,16 +91,10 @@ spec:
           value: quay.io/openshift/origin-csi-driver-shared-resource:latest
         - name: SHARED_RESOURCE_DRIVER_WEBHOOK_IMAGE
           value: quay.io/openshift/origin-csi-driver-shared-resource-webhook:latest
-        - name: ALIBABA_DISK_DRIVER_OPERATOR_IMAGE
-          value: quay.io/openshift/origin-alibaba-disk-csi-driver-operator:latest
-        - name: ALIBABA_CLOUD_DRIVER_IMAGE
-          value: quay.io/openshift/origin-alibaba-cloud-csi-driver:latest
         - name: IBM_VPC_BLOCK_DRIVER_OPERATOR_IMAGE
           value: quay.io/openshift/origin-ibm-vpc-block-csi-driver-operator:latest
         - name: IBM_VPC_BLOCK_DRIVER_IMAGE
           value: quay.io/openshift/origin-ibm-vpc-block-csi-driver:latest
-        - name: IBM_VPC_NODE_LABEL_UPDATER_IMAGE
```

Comment:
This is necessary to remove it?

Reply:
Just following [this](https://github.com/openshift/hypershift/blob/6d862d189fee83a20a2a0a5500f9f9e09bbfdc99/control-plane-operator/controllers/hostedcontrolplane/storage/assets/README.md) and copying contents from [here](https://github.com/openshift/cluster-storage-operator/blob/master/manifests/10_deployment-hypershift.yaml)